tests/typing/test_bounds_tuple.py

```diff
@@ -66,7 +67,9 @@ def pytest_generate_tests(metafunc):
             'UnstructuredGridVolumeRayCastMapper',
             'Volume',
         ]
-        assert sorted(class_names) == sorted(expected_names)
+        diff = set(class_names) - set(expected_names)
+        if diff:
+            pytest.fail(f"The `bounds` property for class {diff}\n must return a {pv.BoundsTuple}.")
```

Comment:
This change is not correct. The aim was to improve the error message, it should say something like:
``` python
f"Class {diff} has a `bounds` property but is not listed as a test case." 
```
I suggest reverting this and leaving this change for another PR.

```suggestion
        assert sorted(class_names) == sorted(expected_names)
```